react/OrderItems.tsx

```diff
@@ -353,10 +357,12 @@ const useUpdateItemsTask = (
       items,
       orderFormItems,
       id,
+      allowedOutdatedData = undefined,
```

Comment:
`undefined` is already the default value for arguments that aren't being passed
```suggestion
      allowedOutdatedData,
```